hello-world-attest-fn-call/README.md

```diff
@@ -0,0 +1,267 @@
+# Hello World
+
+This example shows you how to use the Blocky Attestation Service (Blocky AS) to
+attest a simple function calls.
+
+You'll learn how to:
+
+- Create a function that returns a `"Hello, World!"` message
+- Create a function that writes a message to a log and returns a
+  user-defined error
+- Invoke functions in the Blocky AS using its `bky-as` CLI
+- Extract function output from the Blocky AS attestation
+
+## Setup
+
+- Install the Blocky AS CLI by following the
+  [setup instructions](https://blocky-docs.redocly.app/attestation-service/setup)
+  in the Blocky AS documentation.
+- Make sure you also have
+  [Docker](https://www.docker.com/) and [jq](https://jqlang.org/) installed on
+  your system.
+
+## Quick Start
+
+To run this example, call the following command:
+
+```bash
+make run FUNCTION=hello-world
+```
+
+You will see the following JSON output extracted from a Blocky AS attestation:
+
+```
+Result:
+{
+  "Success": true,
+  "Error": "",
+  "Output": "Hello, World!"
+}
+```
+
+where `Success` indicates whether the function call was successful, `Error`
```

Comment:
The result as described here is a little different than how it is used in the code.

Reply:
removing the result pattern from this example